rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -493,6 +494,12 @@ void log_frame::UpdateUI()
 					m_tty->appendPlainText(tty_text);
 				}
 
+				if (buf.size() == end_line)
```

Comment:
This is kinda strange and is unnecessary in this place anyway.
It would make more sense to integrate this into the if/else inside of the upper m_stack_tty block.

Think about it: you only want to process anything with m_old_tty_text in the stack mode.
And the inner else block also contains the regular text append.
So putting it out here is both wrong logically as well as unoptimized.

Reply:
Removed it for now.